### PR TITLE
fix collection links

### PR DESCRIPTION
**Pull request by tsop14**
Fixes link for non-root collections on preview page and image page.  
Removes one-time data binding as believe it's not possible to have on filtered values. 

https://github.com/angular/angular.js/issues/8605

### CODE REVIEW

**Comment by jamesgorrie** [comment]
Oh well :+1:

**Comment by jamesgorrie** [comment]
Hold on...

**Comment by jamesgorrie** [comment]
Never mind me, being :hatching_chick: :+1:

File: kahuna/public/js/preview/image.html [hunk 1]
```diff
@@ -51,7 +51,7 @@
                     ng:repeat="collection in ctrl.image.data.collections"
                     gr-tooltip="Click to open collection: {{collection.data.path.join(' ▸ ')}}"
                     gr-tooltip-position="top">
-                    <a ui:sref="::(search.results({query: (collections.data.path | queryCollectionFilter)}))"
+                    <a ui:sref="search.results({query: (collection.data.pathId | queryCollectionFilter)})"
```
Comment (theefer) [inline kahuna/public/js/preview/image.html]: Did the `::` disappear on purpose?

Comment (jamesgorrie) [inline kahuna/public/js/preview/image.html]: That's the point of the PR.

Comment (theefer) [inline kahuna/public/js/preview/image.html]: :+1:

**Pull request merged**